server/tapeuse_test.go

```diff
@@ -0,0 +1,205 @@
+package server
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ndlib/bendo/blobcache"
+	"github.com/ndlib/bendo/fragment"
+	"github.com/ndlib/bendo/items"
+	"github.com/ndlib/bendo/store"
+	"github.com/ndlib/bendo/transaction"
+	"github.com/ndlib/bendo/util"
+)
+
+// test /admin/tape_use commmands
+func TestTapeUseAdmin(t *testing.T) {
+	check_status(t, "PUT", "/admin/use_tape/on", 201)
+
+	text := get_body(t, "GET", "/admin/use_tape", 200)
+	if text != "On" {
+		t.Fatalf("Received %#v, expected %#v", text, "On")
+	}
+
+	check_status(t, "PUT", "/admin/use_tape/off", 201)
+
+	text = get_body(t, "GET", "/admin/use_tape", 200)
+	if text != "Off" {
+		t.Fatalf("Received %#v, expected %#v", text, "Off")
+	}
+}
+
+// test Blobs under no_tape_use
+func TestTapeItem(t *testing.T) {
+	// upload a blob, and commit an item
+	check_status(t, "PUT", "/admin/use_tape/on", 201)
+	blob1 := upload_string(t, "POST", "/upload", "hello world")
+	blob2 := upload_string(t, "POST", "/upload", "goodbye cruel world")
+	t.Log("blob1 =", blob1)
+	t.Log("blob2 =", blob2)
+
+	itemid := "zxcvbnm" + randomid()
+	txpath := send_transaction(t, "/item/"+itemid+"/transaction",
+		[][]string{{"add", path.Base(blob1)}, {"slot", "testFile1", path.Base(blob1)}, {"add", path.Base(blob2)}, {"slot", "testFile2", path.Base(blob2)}}, 202)
+	t.Log("got tx path", txpath)
+	// tx is processed async from the commit above.
+	waitTrans(t, txpath)
+
+	t.Log("first")
+	check_status(t, "GET", "/item/"+itemid+"/testFile1", 200)
+	t.Log("second")
+	check_status(t, "GET", "/item/"+itemid+"/testFile1", 200)
+
+	check_status(t, "PUT", "/admin/use_tape/off", 201)
+	// get item cached
+	t.Log("third")
+	check_status(t, "GET", "/item/"+itemid+"/testFile1", 200)
+	// get item non-cached
+	check_status(t, "GET", "/item/"+itemid+"/testFile2", 503)
+}
+
+func TestTapeBundle(t *testing.T) {
+	check_status(t, "PUT", "/admin/use_tape/on", 201)
+	check_status(t, "GET", "/bundle/list/", 200)
+	check_status(t, "GET", "/bundle/list/noone", 200)
+	check_status(t, "GET", "/bundle/open/noone", 200)
+	check_status(t, "PUT", "/admin/use_tape/off", 201)
+	check_status(t, "GET", "/bundle/list/", 503)
+	check_status(t, "GET", "/bundle/list/noone", 503)
+	check_status(t, "GET", "/bundle/open/noone", 503)
+}
+
+//
+// Test Helpers
+//
+
+func upload_string(t *testing.T, verb, route string, s string) string {
```

Comment:
No need to redefine these testing utilities. all of the `*_test` files are compiled into a single binary, and can access functions defined an any of them.